src/journal/journald-native.c

```diff
@@ -88,7 +89,7 @@ void server_process_native_message(
                 const char *label, size_t label_len) {
 
         struct iovec *iovec = NULL;
-        unsigned n = 0, j, tn = (unsigned) -1;
+        unsigned n = 0, j, tn = -1;
```

Comment:
C++ requires a cast, C does not. The way I understand it, the case is not required because it's obvious that if you specify `type x = y`, than you expect the type to be `type`. Anyway, I restored the cast.